packages/tasks/src/local-apps.ts

```diff
@@ -177,6 +177,13 @@ export const LOCAL_APPS = {
 		displayOnModelPage: (model) => model.library_name === "diffusers" && model.pipeline_tag === "text-to-image",
 		deeplink: (model) => new URL(`diffusionbee://open_from_hf?model=${model.id}`),
 	},
+	invoke: {
+		prettyLabel: "Invoke",
+		docsUrl: "https://github.com/invoke-ai/InvokeAI",
+		mainTask: "text-to-image",
+		displayOnModelPage: (model) => model.library_name === "diffusers" && model.pipeline_tag === "text-to-image",
+		deeplink: (model) => new URL(`https://models.invoke.ai/huggingface/${model.id}`),
```

Comment:
nice! 

do you have an example model that's nicely supported by Invoke?

Reply:
Yep! https://huggingface.co/RunDiffusion/Juggernaut-X-v10 (https://models.invoke.ai/huggingface/RunDiffusion/Juggernaut-X-v10)

Reply:
How does deep linking work? On that URL I see a yellow "Install Model" button which links to `http://localhost:9090/api/v2/models/install/huggingface?source=RunDiffusion/Juggernaut-X-v10`, if I click on it I get a 422 error:

`{"detail":[{"type":"int_parsing","loc":["path","id"],"msg":"Input should be a valid integer, unable to parse string as an integer","input":"huggingface"}]}`

Reply:
@pcuenca @julien-c Latest release included the deep link handling if you wouldn't mind taking a look again!

Reply:
<img width="786" alt="Screenshot 2024-08-07 at 10 17 43 AM" src="https://github.com/user-attachments/assets/ef74314a-cad0-4564-b5b0-cef394725f76">

Hi @chainchompa 
I'm still having the same issue while clicking on your live above (https://models.invoke.ai/huggingface/RunDiffusion/Juggernaut-X-v10), then clicking on "Install model"

Reply:
It worked for me after installing locally from `main` 🎉

Got this when clicking on `Install Model`:

<img width="655" alt="Screenshot 2024-08-09 at 19 41 46" src="https://github.com/user-attachments/assets/ea7ec0c0-47a5-4514-a730-46757ec6f90f">

Then a progress indicator in the web UI:

<img width="741" alt="Screenshot 2024-08-09 at 19 42 51" src="https://github.com/user-attachments/assets/210ca602-50ed-4627-a6fd-23fcc96e52cb">

Nit: If you use the same link twice, it will download the model again, as shown in the screenshot above.

Reply:
@enzostvs did you try the Automatic Install? I can test that one if the process failed for you.

Reply:
Ok nvm then, thank you @pcuenca 
Good on my side to merge it 👌